hack/update-codegen.sh

```diff
@@ -34,28 +34,33 @@ export "GOFLAGS=-mod=readonly"
 
 echodate "Generating kubermatic:v1"
 ./vendor/k8s.io/code-generator/generate-groups.sh all \
-  github.com/kubermatic/kubermatic/pkg/crd/client \
-  github.com/kubermatic/kubermatic/pkg/crd \
+  k8c.io/kubermatic/v2/pkg/crd/client \
+  k8c.io/kubermatic/v2/pkg/crd \
   "kubermatic:v1" \
   --go-header-file /tmp/headerfile
 
 echodate "Generating operator:v1alpha1"
 ./vendor/k8s.io/code-generator/generate-groups.sh deepcopy,lister,informer \
-  github.com/kubermatic/kubermatic/pkg/crd/client \
-  github.com/kubermatic/kubermatic/pkg/crd \
+  k8c.io/kubermatic/v2/pkg/crd/client \
+  k8c.io/kubermatic/v2/pkg/crd \
   "operator:v1alpha1" \
   --go-header-file /tmp/headerfile
 
 # Temporary fixes due to: https://github.com/kubernetes/kubernetes/issues/71655
-GENERIC_FILE="pkg/crd/client/informers/externalversions/generic.go"
+GENERIC_FILE="v2/pkg/crd/client/informers/externalversions/generic.go"
 sed -i s/usersshkeys/usersshkeies/g ${GENERIC_FILE}
 
 echodate "Generating deepcopy funcs for other packages"
 go run k8s.io/code-generator/cmd/deepcopy-gen \
-  --input-dirs github.com/kubermatic/kubermatic/pkg/semver \
+  --input-dirs k8c.io/kubermatic/v2/pkg/semver \
   -O zz_generated.deepcopy \
   --go-header-file /tmp/headerfile
 
+# move files into their correct location, generate-groups.sh does not handle
+# non-v1 module names very well
+cp -r v2/* .
+rm -rf v2/
```

Comment:
is this still necessary or inclusion of `k8c.io/kubermatic/v2` in vanitygen solved the issue?

Reply:
I don't understand?

Reply:
Ah, now I understand, this is still required. The updated vanity URL has no impact on this particular hack.

Reply:
If the source code initially placed in `$GOPATH/src/k8c.io/kubermatic/v2`, then codegen works perfectly well, without need for hacks.

Reply:
Yeah but you cannot go-get Kubermatic into a v2 directory. I tried it with a random module:

```
$ go get -u -v github.com/cespare/xxhash/v2
github.com/cespare/xxhash (download)
package github.com/cespare/xxhash/v2: cannot find package "github.com/cespare/xxhash/v2" in any of:
        /usr/local/go/src/github.com/cespare/xxhash/v2 (from $GOROOT)
        /home/xrstf/gospace/src/github.com/cespare/xxhash/v2 (from $GOPATH)

$ go get -u -v github.com/cespare/xxhash
github.com/cespare/xxhash (download)
```

I doubt that we would have to clone Kubermatic into a `v2` directory.

Reply:
```
$ go get -v github.com/cespare/xxhash/v2
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/cespare/xxhash v1.1.0
go: github.com/cespare/xxhash/v2 upgrade => v2.1.1
github.com/cespare/xxhash/v2
```

Reply:
I suppose you had `GO111MODULE=off`

Reply:
But go get places the module into pkg/, not into src/, so how does it relate?